cts/cts-exec.in

```diff
@@ -143,22 +144,26 @@ class ExecTest(Test):
         self._stonith_process = None
 
     def add_cmd(self, args):
+        # pylint: disable=arguments-differ
```

Comment:
I'm not at all happy with this, but aside from doing the obvious thing of making it match the superclass, and then changing every single caller (of which there are many) to have the same argument list, I don't see any other options.  Maybe I should just do that anyway.

Reply:
Updated, but I'm not sure I like what I did there either.  We'll see.